README.md

```diff
@@ -1,73 +1,48 @@
----
-services: active-directory
-platforms: ios
+--- 
+Services: active-directory
+platforms: iOS
 author: brandwe
+level: 100
+client: iOS Mobile App
+service: Microsoft Graph
+endpoint: AAD V2
 ---
```

Comment:
In the description of the repo, update to indicate the sample calls MS Graph. Also, add tags of the sample and what it does.

Reply:
Continuous integration and build badge.